app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -31,7 +26,7 @@ def submit
 
     reset_session
 
-    redirect_to confirmation_url(reference_number: submission_reference)
+    redirect_to confirmation_url(reference_number: submission_reference, contact_gp: contact_gp)
```

Comment:
Could this be `contact_gp?` and you could remove the variable assignment above.

Reply:
No, because the session is reset right above this line, so the session data is already gone.

Reply:
Ah I didn't notice that 👍